test/distributed/test_c10d_functional_native.py

```diff
@@ -499,10 +499,8 @@ def test_inductor_all_gather_into_tensor_single(self):
         self._init_process_group()
 
         def func(arg: torch.Tensor) -> torch.Tensor:
-            ag0 = torch.ops._c10d_functional.all_gather_into_tensor(
-                arg, self.world_size, "default"
-            )
-            ag0 = torch.ops._c10d_functional.wait_tensor(ag0)
+            ag0 = funcol.all_gather_tensor(arg, 0, "default")
```

Comment:
what happened to the 'into', was that intentional?

Reply:
iirc the python api is `funcol.all_gather_tensor`, and the ops binded to the c10d_functional namespace is sth like `all_gather_into_tensor`. Maybe we should make the naming consistent to `all_gather_tensor` in the future